drop-boxes/register-omero-metadata/register-omero.py

```diff
@@ -182,6 +200,21 @@ def getPropertyMap(line, property_names):
 
 	return properties
 
+def filterOmeroPropertyMap(property_map):
+	"""Filters map before ingestion into omero server
+	"""
+
+	#the blacklist, e.g. what is going to openBIS and not to omero
+	filter_list = ["IMAGE_FILE_NAME", "INSTRUMENT_USER", "IMAGING_DATE"]
```

Comment:
Image file name is not going to openBIS at the moment. Is it blacklisted because the file name is not a property for omero either?

Reply:
the filename is automatically added to the omero image by the omero-importer-cli, so we don't really need it here, just to build and pass the file path to the cli